app/assets/stylesheets/application.scss

```diff
@@ -29,3 +29,12 @@ $govuk-font-url-function: "font-url";
   margin-left: 8px;
   vertical-align: middle;
 }
+
+.total-org-stat-results-title {
+  font-size: 63px;
+  padding-right: 54px;
+}
+
+.admin-panel-stats-container {
+  padding-left: 125px;
```

Comment:
This seems a bit weird. Are you using padding to centre the container? If so, you should use something like flexbox instead.